backend/src/models/shiftModel.ts

```diff
@@ -76,4 +76,45 @@ export default class ShiftModel {
               });
           });
      }
+
+     // get all the shift details viewable to a volunteer for a specified month
+     public getShiftsByVolunteerIdAndMonth(volunteer_id: string, month: number, year: number): Promise<any> {
+          return new Promise((resolve, reject) => {
+              const query = `
+                  CALL GetShiftsByVolunteerIdAndMonth(?, ?, ?);
+              `;
+              
+              const values = [volunteer_id, month, year];
+              connection.query(query, values, (error: any, results: any) => {
+                  if (error) {
+                      return reject({
+                          status: 500,
+                          message: `An error occurred while executing the query: ${error}`
```

Comment:
Just gonna add this here for now, but I'm pretty sure this error handling doesn't make sense. Idk why its all over our project, but putting `error` here makes more sense. Not a problem particularly, but maybe smt to note for future endpoints.

Reply:
Don't need to change if its too annoying

Reply:
noted :)